roles/openshift_storage_glusterfs/tasks/glusterfs_common.yml

```diff
@@ -34,6 +34,16 @@
 - import_tasks: glusterfs_deploy.yml
   when: glusterfs_is_native
 
+- name: Start glusterd (external)
```

Comment:
As I understand it, there are 3 deployment options.

1) Native (eg, pod based)
2) Non-native (eg, some RHEL hosts that we install gluster on for the user)
3) External (eg, some gluster that is already setup and we don't touch, just need api/url type info for).

If it's external, we can't manage it?

Reply:
There's only two, native and external. External is assumed to have `glusterfs-server` installed already. All the packages to run basic anisble are available in base CentOS, still need to test if this'll work on RHEL.